packages/design-system/docs/components/OcGrid/OcGrid.md

```diff
@@ -0,0 +1,27 @@
+# OcGrid component
+
+## Description
+
+The `OcGrid` component allows you to arrange block elements in columns.
+
+## Examples
+
+### Default
+
+The component provides a default slot for any content that should be displayed in the grid.
+
+::: livecode
+```html
+<oc-grid>
+	<div style="width: 80px; height: 80px; background-color: var(--oc-color-swatch-passive-default)" class="oc-height-small"></div>
+	<div style="width: 80px; height: 80px; background-color: var(--oc-color-swatch-success-default)" class="oc-height-small"></div>
+	<div style="width: 80px; height: 80px; background-color: var(--oc-color-swatch-warning-default)" class="oc-height-small"></div>
+</oc-grid>
+
+<oc-grid :flex="true" class="oc-mt-m">
```

Comment:
I don't like the `flex` prop we're using here... would be a great chance to change that, since it's only used in one location. Can we replace the `flex` prop with a `direction` prop which is `row` by default and `column` as the second option? That's what flexbox is doing and what this component also does internally, just that `flex` is a super weird prop name for that, especially since it's a boolean.